docs/reference/aggregations/pipeline/movavg-aggregation.asciidoc

```diff
@@ -112,6 +165,7 @@ the values from a `simple` moving average tend to "lag" behind the real data.
     }
 }
 --------------------------------------------------
+// NOTCONSOLE
```

Comment:
I'd love for these to be a running example too. Like if you could take the example above, drop it here, and add this bit. Something like that.